cwms-data-api/src/test/java/cwms/cda/data/dao/texttimeseries/RegularTimeSeriesTextDaoTestIT.java

```diff
@@ -210,4 +209,72 @@ private void testDelete(RegularTimeSeriesTextDao dao) {
 
     }
 
+
+    @Test
+    void testStore() throws SQLException {
+        CwmsDatabaseContainer<?> databaseLink = CwmsDataApiSetupCallback.getDatabaseLink();
```

Comment:
FYI the before all needs to create the location with the base case handler.

Wasn't able to run the test locally due to the location not already existing.